third_party/terraform/resources/resource_google_project_service.go

```diff
@@ -102,6 +103,13 @@ func resourceGoogleProjectService() *schema.Resource {
 	}
 }
 
+// Since we allow the project to now come from config in the form of
+// /projects/${myproject} we need to correct for this when reading the value.
+func adjustProjectReference(project string) string {
```

Comment:
This can be replaced with `GetResourceNameFromSelfLink`

Reply:
Will do